src/azure-cli/install.sh

```diff
@@ -197,6 +198,28 @@ if [ ${#AZ_EXTENSIONS[@]} -gt 0 ]; then
     done
 fi
 
```

Comment:
To tell the `apt` that you've added runs in non-interactive mode

```suggestion
export DEBIAN_FRONTEND=noninteractive
```

Reply:
looks like this line of code was already included on ln#60